### DISCUSSION TITLE
WebUI: Make errors clickable without javascript

### DISCUSSION THREAD

**Discussion by Jellyfrog** [spatie/flareapp.io-roadmap#62] [Ideas]
I prefer to open errors in new tabs by scroll/ctrl-click, but its not possible since the click is handled by javascript and no href is "exposed"
For clarification, one of these boxes:
![image](https://user-images.githubusercontent.com/759887/188420254-a34fa9fd-b3d8-4ac4-b3a8-9b795a75e3ed.png)

**Comment by Sam-Apostel** [comment]
This has been implemented with the redesign.
